dpos/state/arbitrators.go

```diff
@@ -463,11 +465,77 @@ func (a *arbitrators) distributeDPOSReward(height uint32,
 	return
 }
 
-func (a *arbitrators) distributeWithNormalArbitratorsV(height uint32, reward common.Fixed64) (
+func (a *arbitrators) distributeWithNormalArbitratorsV2(height uint32, reward common.Fixed64) (
 	map[common.Uint168]common.Fixed64, common.Fixed64, error) {
 	if len(a.currentArbitrators) == 0 {
 		return nil, 0, errors.New("not found arbiters when " +
-			"distributeWithNormalArbitratorsV")
+			"distributeWithNormalArbitratorsV1")
```

Comment:
should be distributeWithNormalArbitratorsV2

Reply:
the first one is distributeWithNormalArbitratorsV0..